src/qt/forms/optionsdialog.ui

```diff
@@ -714,7 +714,7 @@
         <item>
          <widget class="QLabel" name="overriddenByCommandLineInfoLabel">
           <property name="text">
-           <string>Active command-line options that override above options:</string>
+           <string>GUI options overridden by the command line or in the configuration file:</string>
```

Comment:
I think the first part is confusing, these are not only *GUI* options, but anything that can be configured in the options dialog.

Reply:
Could it be:
"_Options set in this dialog and overridden by the command line or in the configuration file:_" ?

Reply:
yes, that's better